design-proposals/determining-host-model-migratability-level.md

```diff
@@ -0,0 +1,123 @@
+This template is simply a guide. When making proposals, feel free to add and
+remove portions of this outline to best fit your specific case. There is no
+expectation that people strictly stick to this outline, but it is a good place
+to start. 
+
+# Overview
+Add a label to determine the host-model migratability level of each node 
+and use this label to configure the cluster to prefer\require node with
+high host-model migratability level when scheduling vm for the first time.
+## Motivation
+If a vm with host-model cpuModel is scheduled to a node with a unique `host-model-required-feature`
+in heterogeneous cluster it won't be able to migrate and this will probably **SURPRISE** the user.
+
+It isn't a trivial task for a user to calculate the set of nodes that 
+a vm with `host-model` can migrate to (for each node) especially when the cluster change 
+and nodes are being added\removed to\from the cluster.
+
+Even if a user has this information he would have to set nodeAffinity himself to prefer/require
+nodes only for the first time he schedule a vm.
+
+Therefore some users might benefit from such label and configuration options.
+
```

Comment:
Maybe you can also add a motivation for testing. Today we only test homogeneous clusters (partly because of the problem you're raising here), with this label we could extend our tests into a heterogeneous environment easily.

Reply:
Done.